### PR TITLE
docs: Add badges for alternate languages.

### PR DESCRIPTION
**Pull request by darkobits**
Minor tidying-up by creating badges for alternative readme languages.

Consider this a suggestion, and let me know if you think this needs any tweaking.

😄

### CODE REVIEW

**Comment by romainricard** [comment]
I've tried to create badges, but I was not successful. Yours look good! I'm just not sure about the `bit.ly` url, but I see no easy way to shorten the URL to keep the readme clean. :disappointed:

File: README.fr.md [hunk 1]
```diff
@@ -5,12 +5,10 @@
 <p align="center">
   <a href="https://travis-ci.com/signalstickers/signalstickers"><img src="https://travis-ci.com/signalstickers/signalstickers.svg?branch=master"></a>
   <a href="https://github.com/signalstickers/signalstickers/graphs/contributors"><img src="https://img.shields.io/github/contributors/signalstickers/signalstickers"></a>
+  <a href="README.md"><img src="https://bit.ly/31ysLJA" alt="English"></a>
+  <a href="README.de.md"><img src="https://bit.ly/2SnGvCO" alt="Deutsche"></a>
```
Comment (romainricard) [inline README.fr.md]: Extra `e` in `Deutsche`

File: README.md [hunk 2]
```diff
@@ -5,14 +5,12 @@
 <p align="center">
   <a href="https://travis-ci.com/signalstickers/signalstickers"><img src="https://travis-ci.com/signalstickers/signalstickers.svg?branch=master"></a>
   <a href="https://github.com/signalstickers/signalstickers/graphs/contributors"><img src="https://img.shields.io/github/contributors/signalstickers/signalstickers"></a>
+  <a href="README.de.md"><img src="https://bit.ly/2SnGvCO" alt="Deutsche"></a>
```
Comment (romainricard) [inline README.md]: Extra `e` in `Deutsche`

**Comment by darkobits** [comment]
> I've tried to create badges, but I was not successful.

The wonderful [shields.io](https://shields.io) has an API that lets you create custom badges. 😄

**Comment by romainricard** [comment]
I used shields.io, but couldn't find one that I liked :grimacing:

**Pull request merged**